aiocache/backends/memcached.py

```diff
@@ -142,10 +138,16 @@ class MemcachedCache(MemcachedBackend, BaseCache):
     :param pool_size: int size for memcached connections pool. Default is 2.
     """
 
+    NAME = "memcached"
+
     def __init__(self, serializer=None, **kwargs):
         super().__init__(**kwargs)
         self.serializer = serializer or JsonSerializer()
 
+    @classmethod
```

Comment:
This is from before, not sure why I did put the method in the backend class but backend should contain methods only related to the client implementation and this is not